### DISCUSSION TITLE
The abbrs org

### DISCUSSION THREAD

**Discussion by T1xx1** [abbrcode/abbreviations-in-code#94] [Announcements]
As we decided under [issue](https://github.com/abbrcodes/abbreviations-in-code/issues/49) we have created a new org to host our abbr repo [Abbreviations in code](https://github.com/abbrcodes/abbreviations-in-code).

We have not decided on a name for the org yet. The main lines so far are
 - Code-abbrev (suggested by @kisvegabor )
 - AIC-Committee (suggested by @philipeachille )
 - AIC-Consortium (suggested by @philipeachille )
 - Abbrcode (suggested by @philipeachille )
 - Abbreviate codes (suggested by @T1xx1 )
 - Abbre (suggested by @oood)
 - Abbres (suggested by @oood)
 - AbbreviationsInCode (suggested by @T1xx1 )
 - Abbreviations-in-code (suggested by @kisvegabor )
 - Abbreviations for codes (suggested by @T1xx1 )
 - Dictionary for code (suggested by @T1xx1 )
 - The dictionary of abbrs (suggested by @T1xx1 )
 - The organization for Abbreviations in Code (suggested by @T1xx1 )

[Vote](https://github.com/abbrcodes/discussions/discussions/8) a name or feel free to suggest new names.

**Comment by oood** [comment]
@philipeachille

**Comment by oood** [comment]
You guys discuss first, I'm updating [Brevis](https://github.com/orgs/abbrcodes/projects/1/views/1) right now, I honestly like @Brevis, but it's not available as a username.

**Reply by T1xx1** [reply]
I'm also organising the org project moving some tasks from brevis.

**Reply by oood** [reply]
Ok, but, can you do this tomorrow? I need to add a few more, maybe we'll do it tomorrow when I'm done?

**Reply by T1xx1** [reply]
Yes, no problems.

**Reply by oood** [reply]
Ok thanks, as there is still an update to be had there.

**Comment by T1xx1** [comment]
![Untitled-2023-03-10-2100](https://user-images.githubusercontent.com/69322987/224424121-a7646712-a595-4f5c-ad43-45abcbaf6bd5.png)

**Reply by T1xx1** [reply]
![Untitled-2023-03-10-2100](https://user-images.githubusercontent.com/69322987/224425567-66f5e6ec-4c2c-44af-93e4-1b7923bb4689.png)

Better one.

**Reply by philipeachille** [reply]
DB should be within Abbreviations-in-code circle, no?

**Reply by T1xx1** [reply]
It is right now. When the dictionary will be completed there will be another repo with the db and generator scripts.

**Comment by T1xx1** [comment]
Maybe we got a problem.
I've created the [db](https://github.com/abbrcode/db) for abbr-in-code. Right now there's main.abbr.yml, the main file we should maintain for abbr-in-code, and there's also main.abbr.json written in JSON format.
I tried to create the scripts to generate this file (and any other format we may need) but it's not as easy as I thought it could be. I've gen this file using js because I couldn't do it with bash. So I don't know what to do here.
I've thought about 2 sol.
1. Keep these scripts in js. I have them on my pc if you agree to this sol I would commit them.
2. I can try harder on bash.

What do you think?

**Comment by T1xx1** [comment]
I was formatting alexanderpann's list and I'm finding a lot of acronyms in it.
While thinking about it I searched over abbrincode to see if there were any, because I noticed that acronyms strictly require at least two words to form and in abbrincode there are some.
We added some acronyms still they couldn't be in there.

I searched the exact def of acro and...

> "An acronym is a kind of abbreviation. Abbreviations can be shortened forms of any kind. For example, appt is an abbreviation of appointment, and ASAP is an abbreviation of as soon as possible."

Seems that acro are abbrs.

[src](https://www.dictionary.com/e/acronym-vs-abbreviation/)
> "Specifically, an [acronym](https://www.dictionary.com/browse/acronym) is a specific type of abbreviation"

They are a particular case of abbr.

So I think we should accept acronyms in the end in abbrincode starting from alexanderpann ones.
